data/transaction/apiTransactionResult.go

```diff
@@ -59,6 +59,12 @@ type ApiTransactionResult struct {
 	InitiallyPaidFee                  string                    `json:"initiallyPaidFee,omitempty"`
 	IsRelayed                         bool                      `json:"isRelayed,omitempty"`
 	IsRefund                          bool                      `json:"isRefund,omitempty"`
+	CallType                          int                       `json:"callType,omitempty"`
+	RelayerAddress                    string                    `json:"relayerAddress,omitempty"`
+	RelayedValue                      string                    `json:"relayedValue,omitempty"`
+	ChainID                           string                    `json:"chainID"`
+	Version                           uint32                    `json:"version,omitempty"`
+	Options                           uint32                    `json:"options,omitempty"`
```

Comment:
`options` is `0` for most transactions, therefore this field will be excluded. However, SCRs won't have this value set. What should we do in such cases?

A suggestion would be making it an optional uint32 ? {struct { Value uint32, HasValue bool } }

Reply:
I think it is fine to be omitted every time it is set to 0.
It can be interpreted like: "there is no option set => no option has been used; I will only consider this field when is != 0/not omoted"

Reply:
After discussing with @ccorcoveanu, we decided to remove `omitempty`